doc/security/sensor-threat.rst

```diff
@@ -49,23 +49,23 @@ included in this model:
    for updates to this image are:
 
    a. The image shall only be replaced with an authorized image
-      [th-authrepl]_.
+      [#th-authrepl]_.
 
    b. When an authorized replacement image is available, the update
-      shall be done in a timely manner [th-timely-update]_.
+      shall be done in a timely manner [#th-timely-update]_.
 
    c. The image update shall be seen as atomic, meaning that when the
       image is run, the flash shall contain either the update image in
       its entirety, or the old image in its entirety
-      [th-atomic-update]_.
+      [#th-atomic-update]_.
 
 3. **Root certificate list**. In order to authenticate the cloud service
    (server), the IoT device must have a list of root certificates that
    are allowed to sign the certificate on the server. For cloud-provider
    based services, this list will generally be provided by the service
    provider. Because the root certificates can expire, and possibly be
    revoked, this list will need to be periodically updated
-   [th-root-certs]_, [th-root-check]_.
+   [#th-root-certs]_:sup:`,` [#th-root-check]_.
```

Comment:
I'd rather remove the comma here -- forcing it to render as superscript is making too strong of an assumption on how the footnote link will render.

```suggestion
   [#th-root-certs]_ [#th-root-check]_.
```

Reply:
Removed.